### ISSUE TITLE
[SUGGESTION] Add setting to disable hatch animations

### ISSUE THREAD

**Issue by SpenserJ** [pokeclicker/pokeclicker#875]
Hatch animations should be bound to a setting so that they can be disabled if the user does not want them. The [WCAG](https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html) guidelines for accessible sites recommends allowing automatic animations to be stopped to assist users that find them distracting.

**Current Behavior**

Animations play automatically when an egg is 70% ready to hatch
